.bazelignore

```diff
@@ -0,0 +1 @@
+e2e/
```

Comment:
ah, sorry, still one thing missing - the CI isn't running for the new e2e folder.

Could you update like this? https://github.com/aspect-build/rules_terser/commit/a924332479edad569839b1eab2be8651c140bf8d#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133dd

Reply:
That was really difficult, but this should all be good now.

I'm not sure why the GitHub Actions runner has such an issue with the Deno cache in its default location, but it's probably better to create our own cache for the Bazel toolchain. It shouldn't really matter, but if the system has Deno installed, sharing cache with that will remove some hermeticity.

Reply:
yeah I understand this was hard, thank you for figuring it out, since that would be a problem for users as well!